Dockerfiles/midstream/run_tests.py

```diff
@@ -6,43 +6,62 @@
 import subprocess
 from os import path
 
+OS_CWD = os.getcwd()
+# global variables set 
+OPERATOR_DIR=OS_CWD+"/operator_dir/"
+OPERATOR_WORK_DIR=OS_CWD+"/test_operator_work_dir/"
+WORK_DIR=OS_CWD+"/output/"
+PLAYBOOKS_DIR="/operator-test-playbooks/"
+ERROR_MESSAGE_PATH=OS_CWD+"/.errormessage"
 
 class RunMidstreamCVPTests():
 
     def setUp(self):
-        self.operator_dir = os.getenv('OPERATOR_DIR',
-                                      "/project/operator_dir/")
-        self.operator_work_dir = os.getenv('OPERATOR_WORK_DIR',
-                                           "/project/test_operator_work_dir/")
-        self.work_dir = os.getenv('WORK_DIR',
-                                  "/project/output/")
-        self.playbooks_dir = os.getenv('PLAYBOOKS_DIR',
-                                       "/project/operator-test-playbooks/")
+        # IMAGE_TO_TESTis an environment variable that takes 
+        # operator-bundle-image 
         self.image_to_test = os.getenv('IMAGE_TO_TEST')
-        self.umoci_bin_path = os.getenv('UMOCI_BIN_PATH',
-                                        '/usr/local/bin/umoci')
+        # VERBOSITY is verbosity of output log range varying from 1 to 4
+        self.verbosity = int(os.getenv('VERBOSITY', '1'))*"v"
+
+    def run_subprocess_command(self, exec_cmd):
+        print(exec_cmd)
+        result = {}
+        with subprocess.Popen(exec_cmd,
+                              stdout=subprocess.PIPE,
+                              stderr=subprocess.PIPE,
+                              shell=True) as process:
```

Comment:
-1 for `subprocess.Popen` it was necesary jsut for python2/3 compatibility

`subprocess.run` is modern and better replacement